C/Cpp_include/c4ConnectedClient.hh

```diff
@@ -25,7 +26,26 @@ struct C4ConnectedClient  : public fleece::RefCounted,
     static Retained<C4ConnectedClient> newClient(const C4ConnectedClientParameters &params);
                     
     /// Gets the current revision of a document from the server.
-    virtual litecore::actor::Async<C4DocResponse> getDoc(C4Slice, C4Slice, C4Slice, bool) noexcept=0;
+    virtual litecore::actor::Async<C4DocResponse> getDoc(slice,
+                                                         slice,
+                                                         slice,
+                                                         bool) noexcept=0;
+    
+    /// Pushes a new document revision to the server.
+    /// @param docID  The document ID.
+    /// @param collectionID  The name of the document's collection, or `nullslice` for default.
+    /// @param revID The new revision ID
+    /// @param parentRevID The ID of the parent revision on the server,
+    ///                      or `nullslice` if this is a new document.
+    /// @param revisionFlags  Flags of this revision.
+    /// @param fleeceData  The document body encoded as Fleece (without shared keys!)
+    /// @return  An async value that, when resolved, contains the status as a C4Error.
+    virtual litecore::actor::Async<void> putDoc(slice docID,
+                                                slice collectionID,
+                                                litecore::revid revID,
+                                                slice parentRevID,
+                                                litecore::Rev::Flags revisionFlags,
```

Comment:
The public type is C4RevisionFlags. (Just look at the types used by the underlying ConnectedClient method.)